src/main/java/com/example/demo/model/User.java

```diff
@@ -0,0 +1,176 @@
+package com.example.demo.model;
+
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Optional;
+
+import javax.persistence.CascadeType;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.validation.constraints.Email;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+
+@Entity
+public class User implements Serializable {
+	
+	@Id
+	@GeneratedValue(strategy= GenerationType.AUTO)
+	private int id;
+	private String name;
+	private String password;
+	
+	@OneToOne(targetEntity=Address.class,cascade=CascadeType.ALL,mappedBy="user")
+	@JsonIgnoreProperties("user")
+	private Address address;
+	
+	@Email(regexp="^[A-Za-z0-9+_.-]+@(.+)$")
+	private String email;
+	
+	private String accountState;
+	
+	@ElementCollection
```

Comment:
Use of @Elementcollection?

Reply:
Embedding list in a User table is creating the issue. 
So, with this annotation it created new table itself and mapped it with User in database